odl/tomo/geometry/fanbeam.py

```diff
@@ -58,51 +61,167 @@ def __init__(self, apart, dpart, src_radius, det_radius, **kwargs):
         src_to_det_init : `array-like` (shape ``(2,)``), optional
             Initial state of the vector pointing from source to detector
             reference point. The zero vector is not allowed.
-            Default: ``(1, 0)``.
-        det_init_axis : `array-like` (shape ``(2,)``), optional
-            Initial axis defining the detector orientation.
-            By default, a normalized `perpendicular_vector` to
-            ``src_to_det_init`` is used.
+            Default: ``(-1, 0)``.
+        det_axis_init : `array-like` (shape ``(2,)``), optional
+            Initial axis defining the detector orientation. The default
+            depends on ``src_to_det_init``, see Notes.
+        extra_rot : `array_like`, shape ``(2, 2)``, optional
+            Rotation matrix that should be applied at the end to the
+            configuration of ``src_to_det_init`` and ``det_axis_init``.
+            The rotation is extrinsic, i.e., defined in the "world"
+            coordinate system.
+
+        Notes
+        -----
+        In the default configuration, the initial source-to-detector vector
+        is ``(1, 0)``, and the initial detector axis is ``(0, 1)``. If a
+        different ``src_to_det_init`` is chosen, the new default axis is
+        given as a rotation of the original one by a matrix that transforms
+        ``(1, 0)`` to the new (normalized) ``src_to_det_init``. This matrix
+        is calculated with the `rotation_matrix_from_to` function.
+        Expressed in code, we have ::
+
+            init_rot = rotation_matrix_from_to((1, 0), src_to_det_init)
+            det_axis_init = init_rot.dot((0, 1))
+
+        Examples
+        --------
+        Initialization with default parameters and some radii:
+
+        >>> e_x, e_y = np.eye(2)  # standard unit vectors
```

Comment:
Nothing wrong in having both examples in the code, but we should at least have one very simple usage example